### ISSUE TITLE
Pitch Bends

### ISSUE THREAD

**Issue by motifn** [Tonejs/MidiConvert#80]
Pitch bends don't appear to be supported. Would expect something like:

```
addPitchBend (pb: Partial<PitchBendInterface>) {...}
```

Looking at the underlying midi-file library, it seems pitch bends are encoded with a type of "pitchBend", but apart from that, the pattern is very similar to other midi controller events. So the encoding translation might look something like:

```
function encodePitchBend (pb: PitchBend, channel: number): MidiPitchBendEvent {
	return {
		absoluteTime: pb.ticks,
		channel,
		deltaTime: 0,
		type: "pitchBend",
		value: pb.value,
	};
}
```

**Comment by tambien** [comment]
Seems like the right API. PRs welcome!

**Comment by motifn** [comment]
See pull request. Note that I didn't add any tests, but see some existing tests are failing for presumably passive reasons (where the mere presence of the new `pitchBends` field on `Track` makes existing tests fail). Should be straightforward for you to address.

Also, one simple tweak you could make to the pull request: rather than set the raw pitch value, set it like this:

```
value: Math.floor(pb.value * 8191.5)
```

This will translate incoming values where:
 * `-1` = the maximum lowest bend
 * `1` = maximum highest bend
 * `0` = no bend

Then, whether you additionally want to truncate values not between -1 and 1,or error? Your call.

Haven't considered setting pitch wheel ranges; maybe leave that as a future feature for now?

References:

The underlying midi-file implementation:

```
	 case 'pitchBend':
	      eventTypeByte = 0xE0 | event.channel
	      if (eventTypeByte !== lastEventTypeByte) w.writeUInt8(eventTypeByte)
	      var value14 = 0x2000 + event.value
	      var lsb14 = (value14 & 0x7F)
	      var msb14 = (value14 >> 7) & 0x7F
	      w.writeUInt8(lsb14)
	      w.writeUInt8(msb14)
```
An article explaining the pitch bend encoding:
https://sites.uci.edu/camp2014/2014/04/30/managing-midi-pitchbend-messages/

**Comment by tambien** [comment]
closed by #82
